src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
+import TimeStamp from './TimeStamp';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const changeMyLike = () => {
+    props.likedCallback(props.id);
+  };
+
+  const messageFormat = props.sender === 'Estragon' ? 'remote' : 'local';
```

Comment:
Tip: This is a little brittle, since it means we can't re-use these components for a chat between other people.